static/scripts/app-register.js

```diff
@@ -54,119 +54,120 @@ formbtn.addEventListener('click', async (e) => {
     
     const formInputs = document.querySelectorAll('.form-inputs');
     const userNameInput = document.querySelector('#username');
-    const userEmailInput = document.querySelector('#email');
-    const userPassInput = document.querySelector('#password');
-
-      let validation = preValidation(formInputs, REGEXP);
-
-      if(validation.every(el => el === false)){
-        const fv = new FormData(formValues);
-        const obj = Object.fromEntries(fv);
-        const jsonData = JSON.stringify(obj);
-        console.log("sending JSON:", jsonData)
-        const datafetch = await fetch('/register', {
-            method: 'POST',
-            headers: {
-                'Content-Type': 'application/json;charset=utf-8'
-              },
-            body: jsonData
-         });
-    
-        let result = await datafetch.json();
 
-        if(datafetch.ok) {
-            console.log(datafetch.status)
-            console.log(datafetch.statusText)
-        } else if(datafetch.status === 409){
-            
+    let validation = preValidation(formInputs, REGEXP);
+    if(!validation.every(el => el === false)) {
+        console.log('no validation happening');
+        return;
+    }
+    // Validation & comms to user
+    const fv = new FormData(formValues);
+    const obj = Object.fromEntries(fv);
+    const jsonData = JSON.stringify(obj);
+    console.log("sending JSON:", jsonData)
+    const datafetch = await fetch('/register', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json;charset=utf-8'
+            },
+        body: jsonData
+        });
+
+    switch (datafetch.status) {
+        case 200:
+            console.log('successful registration');
+            // TODO @mike implement better comms to user
```

Comment:
how? add some green banner with text like 'SUCCESS' ??

Reply:
Yeah, some sort of UI "sugar" that will inform the clientele that the operations have succeeded.